addons/phone_validation/models/res_partner.py

```diff
@@ -2,18 +2,31 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, models
+from odoo.addons.phone_validation.tools import phone_validation
 
 
 class Partner(models.Model):
     _name = 'res.partner'
-    _inherit = ['res.partner', 'phone.validation.mixin']
+    _inherit = ['res.partner']
 
     @api.onchange('phone', 'country_id', 'company_id')
     def _onchange_phone_validation(self):
         if self.phone:
-            self.phone = self.phone_format(self.phone)
```

Comment:
Can't call "phone_get_sanitized_number" for this one to avoid re-defining the method?
I had done it in my commit: https://github.com/odoo/odoo/pull/40032/commits/e811d59817ab630f24df81a01e602bcefd5e75c6#diff-9f9af87c803d2fe1d849e9f333ba5e83b232c0e7c6a52a00ce290b99f21d4479R14

But maybe it was not correct for some reason?

Reply:
Because res.partner does not inherit form mail.thread.phone (done in SMS, not here).

Reply:
Maybe it would be easier to make it inherit here then and avoid re-implementing the method?

Reply:
Well I am not sure. Probably to be done someday when cleaning mail.thread.phone . But it implies a bit more in term of db and migration.